ApplicationPattern.yaml

```diff
@@ -5418,6 +5422,368 @@ paths:
                                                   - 'operation-client-interface-1-0:LIFE_CYCLE_STATE_TYPE_OBSOLETE'
                                                   - 'operation-client-interface-1-0:LIFE_CYCLE_STATE_TYPE_UNKNOWN'
                                                   - 'operation-client-interface-1-0:LIFE_CYCLE_STATE_TYPE_NOT_YET_DEFINED'
+                                  - description: 'elasticsearch client'
+                                    type: object
+                                    required:
+                                      - local-id
+                                      - layer-protocol-name
+                                      - elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac
+                                    properties:
+                                      local-id:
+                                        type: string
+                                      layer-protocol-name:
+                                        type: string
+                                        enum:
+                                          - 'operation-server-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_OPERATION_LAYER'
+                                          - 'http-server-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_HTTP_LAYER'
+                                          - 'tcp-server-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_TCP_LAYER'
+                                          - 'operation-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_OPERATION_LAYER'
+                                          - 'elasticsearch-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_ELASTICSEARCH_LAYER'
+                                          - 'http-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_HTTP_LAYER'
+                                          - 'tcp-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_TCP_LAYER'
+                                      elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac:
+                                        type: object
+                                        required:
+                                          - elasticsearch-client-interface-configuration
+                                          - elasticsearch-client-interface-status
+                                        properties:
+                                          elasticsearch-client-interface-configuration:
+                                            type: object
+                                            required:
+                                              - auth
+                                              - index-alias
+                                              - service-records-policy
```

Comment:
service-records-policy might not be required - it would not be defined when an app needs to just create and use one index